fabric-screen-handler-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,30 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-screen-handler-api-v1",
+  "name": "Fabric Screen Handler API (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-screen-handler-api-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.8.0",
+    "fabric-api-base": "*"
```

Comment:
Since I see the `fabric-networking-v0` in the build.gradle, care putting it here as well

Reply:
Done.